packages/babel-plugin-transform-function-name/test/fixtures/function-name/modules-2/input.mjs

```diff
@@ -1,4 +1,4 @@
-import last from "lodash/last"
+import last from "lo-dash/last"
```

Comment:
This is just an input/output fixture, but it was annoying when grepping for `lodash` :joy:

Reply:
lol we want to ctrl+f and find no results